lib/lita/handlers/lunch_reminder.rb

```diff
@@ -41,6 +41,9 @@ def load_on_start(_payload)
           karma_for_new_user = @karmanager.average_karma(@assigner.lunchers_list)
           @karmanager.set_karma(user.id, karma_for_new_user)
           response.reply("Le asigne #{karma_for_new_user} a #{user.mention_name} con id #{user.id}")
+          broadcast_to_audit_channel("Se empezó a considerar a #{user.mention_name}. " +
+            "Nuevo karma:  #{karma_for_new_user}"
+          )
```

Comment:
Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.